CheckDrive.Api/CheckDrive.Application/DTOs/Identity/RegisterDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CheckDrive.Domain.Enums;
+using System.ComponentModel.DataAnnotations;
+
+namespace CheckDrive.Application.DTOs.Identity;
+
+public record RegisterDto(
+    string Username,
+    string Password,
+    string PasswordConfirm,
+    string PhoneNumber,
+    string? Email,
+    string FirstName,
+    string LastName,
+    string Address,
+    string Passport,
+    DateTime Birthdate);
```

Comment:
you can add **Position** property so that we can reuse it in **AccountsController** for manager's to create new users.